mlir/include/mlir/Dialect/Vector/Transforms/VectorDistribution.h

```diff
@@ -43,7 +43,9 @@ void populateWarpExecuteOnLane0OpToScfForPattern(
 using DistributionMapFn = std::function<AffineMap(Value)>;
 
 /// Distribute transfer_write ops based on the affine map returned by
-/// `distributionMapFn`.
+/// `distributionMapFn`. Writes of size more than `maxNumElementToExtract`
+/// will not be distributed.
```

Comment:
It might be helpful to document how users are expected to choose a value for this. I could see most users just setting it to `subgroupSize - 1` because might as well not distribute this vector and try to get other distribution opportunities, rather than just failing (or something like that, just thinking out loud).

Reply:
Thanks, added